crates/store/re_server/src/store.rs

```diff
@@ -299,4 +485,100 @@ impl InMemoryStore {
     pub fn iter_datasets(&self) -> impl Iterator<Item = &Dataset> {
         self.datasets.values()
     }
+
+    pub fn table(&self, entry_id: EntryId) -> Option<&Table> {
+        self.tables.get(&entry_id)
+    }
+
+    pub fn table_mut(&mut self, entry_id: EntryId) -> Option<&mut Table> {
+        self.tables.get_mut(&entry_id)
+    }
+
+    pub fn table_by_name(&self, name: &str) -> Option<&Table> {
+        let entry_id = self.id_by_name.get(name).copied()?;
+        self.table(entry_id)
+    }
+
+    pub fn iter_tables(&self) -> impl Iterator<Item = &Table> {
+        self.tables.values()
+    }
+}
+
+macro_rules! generate_entries_table {
```

Comment:
Could this be turned into a regular method? It seems to me that all it'd take is to have  `as_entry_details()` behind a trait for `Table` and `Dataset`.